fiftyone/core/cli.py

```
@@ -4194,6 +4195,100 @@ def execute(parser, args):
             fop.delete_plugin(name)
 
 
+class LabsCommand(Command):
+    """Tools for working with FiftyOne Labs."""
+
+    @staticmethod
+    def setup(parser):
```

Ruff (0.14.7)

Missing return type annotation for staticmethod `setup`

Add return type annotation: `None`

(ANN205)